dask/dataframe/core.py

```diff
@@ -3939,6 +3973,21 @@ def map(self, arg, na_action=None, meta=no_default, is_monotonic=False):
             applied = applied.clear_divisions()
         return applied
 
+    @property
+    @derived_from(pd.Index)
+    def is_monotonic(self):
+        return super().is_monotonic_increasing
```

Comment:
The reason to have this is so that we can use the Index docstring right?

Reply:
Yes, that is correct.